<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>802</width>
    <height>600</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Projects</string>
  </property>
  <widget class="QPushButton" name="backBtn">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>0</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Go Back</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableView">
   <property name="geometry">
    <rect>
     <x>-460</x>
     <y>40</y>
     <width>1901</width>
     <height>821</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Project Name    :</string>
   </property>
  </widget>
  <widget class="QLabel" name="projectName">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>70</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Project</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Description        :</string>
   </property>
  </widget>
  <widget class="QLabel" name="projectDescription">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>100</y>
     <width>541</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Project</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Formula             :</string>
   </property>
  </widget>
  <widget class="QLabel" name="formula">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>140</y>
     <width>541</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>No data found yet</string>
   </property>
  </widget>
  <widget class="QLabel" name="compval_1">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>321</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="compval_2">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>180</y>
     <width>321</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="compval_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>220</y>
     <width>321</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="compval_4">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>220</y>
     <width>321</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="compval_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>260</y>
     <width>321</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="compval_6">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>260</y>
     <width>321</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="compval_7">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>300</y>
     <width>321</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="compval_8">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>300</y>
     <width>321</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="compval_9">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>340</y>
     <width>321</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="compval_10">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>340</y>
     <width>321</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="compval_11">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>380</y>
     <width>321</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="compval_12">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>380</y>
     <width>321</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="compval_13">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>430</y>
     <width>321</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="compval_14">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>420</y>
     <width>321</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="homeBtn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>81</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Home</string>
   </property>
   <property name="icon">
    <iconset theme="../../static/icons/home.png">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>18</width>
     <height>18</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>530</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Export</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exportPdf">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>530</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>pdf</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exportDoc">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>530</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>word</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exportXls">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>530</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>excel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
